src/mongo/guild.js

```diff
@@ -1,10 +1,10 @@
-const {model, Schema} = require('mongoose');
-const { prefixes } = require('../../config/config.json');
+const { model, Schema } = require('mongoose')
+const { prefixes } = require('../../config/config.json')
 
-let guildsSchema = new Schema({
-	ID: {type: String},
-	Name: {type: String},
-	Prefix: {type: String, default: prefixes}
+const guildsSchema = new Schema({
+  ID: { type: String },
+  Name: { type: String },
+  Prefix: { type: String, default: prefixes }
 })
 
-modole.exports = model("guilds", guildsSchema)
\ No newline at end of file
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Shibe-Studio/Shynk/pullRequest?prid=7543351)
```suggestion
modole.exports = model("guilds", guildsSchema)
```